### ISSUE TITLE
Question for Access to LLaVA-1.6 Data

### ISSUE THREAD

**Issue by patrick-tssn** [EvolvingLMMs-Lab/LongVA#10]
Thank you for your excellent work! Since the [LLaVA-1.6](https://llava-vl.github.io/blog/2024-01-30-llava-next/) is currently unavailable, could you please let us know how we can access the data? Is it possible for you to share it with us?

**Comment by patrick-tssn** [comment]
Is your data the same as https://huggingface.co/datasets/lmms-lab/LLaVA-NeXT-Data ?

**Comment by kcz358** [comment]
> Is your data the same as https://huggingface.co/datasets/lmms-lab/LLaVA-NeXT-Data ?

Yes we use the same instruction tuning data in LLaVA-1.6. The only difference is that we can't open-source the 15k user data but everything else is the same in this link

**Comment by patrick-tssn** [comment]
@kcz358 Thank you for your reply. I am currently facing some challenges while trying to reproduce the results of LLaVA-NeXT combined with a long-context LLM. Could you please provide an ablation study for the closed-source 15k data? I am curious about the extent to which this dataset impacts the overall results.

**Comment by kcz358** [comment]
Hi, sorry but we haven't perform ablation on this, so we also don't know what might be the reason for this and how this 15k data may effect the overall performance.
